### PR TITLE
Add GEL Group A breakpoint

### LINKED ISSUE #348: Add GEL Group A breakpoint to gel-foundations
**Is your feature request related to a problem? Please describe.**
gel-foundations does not provide a constant for the Group A breakpoint

**Describe the solution you'd like**
Add a breakpoint constant for 0 - 319 pixels in Group A as specified here http://www.bbc.co.uk/gel/guidelines/typography. There should be constants for the GEL_GROUP_A_MIN_WIDTH and GEL_GROUP_A_MAX_WIDTH boundaries of GROUP A and a media query constant that comprises both (name TBD)

**Describe alternatives you've considered**
No alternatives

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

- [ ] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #348

**Overall change:** Added breakpoint for missing GEL Group A

**Code changes:**

- Added new `GEL_GROUP_A_MAX_WIDTH` constant
- Added new `FEATURE_PHONE_ONLY` breakpoint for 0 - 319 pixels
---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

**Review by dr3 — CHANGES_REQUESTED** [review]
Hey, Apologies, accidentally approved before, this needs change log, package bump etc :)

File: packages/utilities/gel-foundations/src/breakpoints.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
    These namings are based on the GEL description. They are also known as group B and group D
```
Comment (sareh) [inline packages/utilities/gel-foundations/src/breakpoints.js]: Could you please increase the version number in the `package.json` and `package-lock.json` & also add a line in the `CHANGELOG.md`?

**Review by sareh — CHANGES_REQUESTED** [review]
Looks good! Just some package admin steps needed.

**Comment by DenisHdz** [comment]
Thank you for reminding about the versions. I have to get used to it :)

File: packages/utilities/gel-foundations/CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-|---------|-------------|
-| 0.2.4   | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
-| 0.2.3   | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
-| 0.2.2   | [PR#304](https://github.com/BBC-News/psammead/pull/304) Add font family definitions, which include GEL-defined font fallbacks. |
-| 0.2.1   | [PR#303](https://github.com/BBC-News/psammead/pull/303) Bump to ensure correct build is published |
-| 0.2.0   | [PR#290](https://github.com/BBC-News/psammead/pull/290) Move typography font-sizes to use rem instead of em |
-| 0.1.4   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
-| 0.1.1   | [PR#212](https://github.com/BBC-News/psammead/pull/212) Update package description and README. |
+| Version | Description                                                                                                                                                  |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (dr3) [inline packages/utilities/gel-foundations/CHANGELOG.md]: Looks like this file got caught by an auto-linter, can you reset the formatting please? 🙈

**Review by sareh — APPROVED** [review]
👍 Looks great!

**Review by dr3 — APPROVED** [review]

**Comment by dr3** [comment]
@DenisBBC Is this PR not in a project? I cant see one attached

Also `I have assigned myself to this PR and the corresponding issues` has been ticked but you havent 🙈

**Comment by DenisHdz** [comment]
> @DenisBBC Is this PR not in a project? I cant see one attached
> 
> Also `I have assigned myself to this PR and the corresponding issues` has been ticked but you havent 🙈

Yes it is, for some reason it wasn't assigned to the WS Off-Forge project.

**Pull request merged**